### PR TITLE
Support ec2 dhcp options

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description
This change enables support for dhcp_option(s). Includes both singular & plural resource

### Issues Resolved

https://github.com/inspec/inspec-aws/issues/302
https://github.com/inspec/inspec-aws/issues/306

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [X] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [X] Documentation provided or updated for resources 
- [X] All Integration Tests pass
- [X] All Unit Tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/aws_ec2_dhcp_options.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: About the aws_ec2_dhcp_options Resource
+platform: aws
+---
+
+# aws\_ec2\_dhcp\_options
+
+Use the `aws_ec2_dhcp_options` InSpec audit resource to test properties of a single AWS DHCP Options.
```
Comment (IanMadd) [inline docs/resources/aws_ec2_dhcp_options.md:8]: Is this testing one set of options, or multiple sets of options?

File: docs/resources/aws_ec2_dhcp_option.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    describe aws_ec2_dhcp_option(dhcp_options_id: 'dopt-0123456789abcdefg') do
+      it { should exist }
+    end
+
+    describe aws_ec2_dhcp_option(name: 'dopt-vpc-1') do
+        it { should exist }
+    end
+
+#### Parameters
+This resource accepts a one of the below mentioned parameters
```
Comment (IanMadd) [inline docs/resources/aws_ec2_dhcp_option.md:27]: Is one of these required?

Comment (sathish-progress) [reply docs/resources/aws_ec2_dhcp_option.md:27]: Yes one of them is mandatory!

File: docs/resources/aws_ec2_dhcp_options.md (hunk 1 above)
Comment (sathish-progress) [reply docs/resources/aws_ec2_dhcp_options.md:8]: Multiple Set of Options actually!

**Review by Nirbhay1997 — APPROVED** [review]
LGTM

**Pull request merged**